### ISSUE TITLE
module 'mps_youtube.pafy' has no attribute 'video_search' on album matching

### ISSUE THREAD

**Issue by AndreiBiziuk** [iamtalhaasghar/yewtube#63]
## Issue / Suggestion

When using album command, on the matching step getting error:
>Error fetching data. Possible network issue.
module 'mps_youtube.pafy' has no attribute 'video_search'

## Your Environment

>-> % yt --version                                            
yewtube version    : 2.8.1
yt_dlp version     : 2022.02.04
Python version     : 3.10.2 (main, Jan 15 2022, 19:56:27) [GCC 11.1.0]
Processor          : 
Machine type       : x86_64
Architecture       : 64bit, ELF
Platform           : Linux-5.16.13-arch1-1-x86_64-with-glibc2.35
sys.stdout.enc     : utf-8
default enc        : utf-8
Config dir         : /home/andrei/.config/mps-youtube
env:TERM           : xterm-256color
env:SHELL          : /bin/zsh
env:LANG           : ru_RU.UTF-8
env:LANGUAGE       : ru:en_US

**Comment by iamtalhaasghar** [comment]
@AndreiBiziuk 
Uninstall previous version and install latest git version as shown [here](https://github.com/iamtalhaasghar/yewtube#latest-version).

**Comment by AndreiBiziuk** [comment]
I've uninstalled yewtube
>pipx uninstall yewtube

and installed latest version
>pipx install git+https://github.com/iamtalhaasghar/yewtube.git

now every search or opening of playlist crashes with errors:

> ERROR: iuYxW_RiSaw: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> Exception in thread Thread-2 (_preload):
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 617, in extract
>     ie_result = self._real_extract(url)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 3575, in _real_extract
>     info['upload_date'] = strftime_or_none(self._extract_time_text(vpir, 'dateText')[0], '%Y%m%d')
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 732, in _extract_time_text
>     self._search_regex(
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 1184, in _search_regex
>     return next(g for g in mobj.groups() if g is not None)
> StopIteration
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1389, in wrapper
>     return func(self, *args, **kwargs)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1459, in __extract_info
>     ie_result = ie.extract(url)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 647, in extract
>     raise ExtractorError('An extractor error has occurred.', cause=e, video_id=self.get_temp_id(url))
> yt_dlp.utils.ExtractorError: iuYxW_RiSaw: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
>     self.run()
>   File "/usr/lib/python3.10/threading.py", line 946, in run
>     self._target(*self._args, **self._kwargs)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/streams.py", line 189, in _preload
>     streamlist = get(song)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/streams.py", line 54, in get
>     ps = pafy.get_video_streams(ytid)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/pafy.py", line 38, in get_video_streams
>     info_dict = ydl.extract_info(ytid, download=False)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1380, in extract_info
>     return self.__extract_info(url, self.get_info_extractor(ie_key), download, extra_info, process)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1407, in wrapper
>     self.report_error(str(e), e.format_traceback())
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 939, in report_error
>     self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 879, in trouble
>     raise DownloadError(message, exc_info)
> yt_dlp.utils.DownloadError: ERROR: iuYxW_RiSaw: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U

**Comment by iamtalhaasghar** [comment]
@AndreiBiziuk
please report this issue on
 https://github.com/yt-dlp/yt-dlp

**Comment by AndreiBiziuk** [comment]
It's weird, but i have yt-dlp version 2022.03.08.2, despite most recent version on git is 2022.03.08.1 and in "Broken site" issue template there is checklist with "I've verified that I'm running yt-dlp version **2022.03.08.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) *" checkbox.

pip and pipx installs yt-dlp 2022.03.08.2 automatically

>yt --version
yewtube version    : 2.8.1
yt_dlp version     : 2022.03.08.2
Python version     : 3.10.2 (main, Jan 15 2022, 19:56:27) [GCC 11.1.0]
Processor          : 
Machine type       : x86_64
Architecture       : 64bit, ELF
Platform           : Linux-5.16.14-arch1-1-x86_64-with-glibc2.35
sys.stdout.enc     : utf-8
default enc        : utf-8
Config dir         : /home/andrei/.config/mps-youtube
dbus               : None
glib               : False
env:TERM           : xterm-256color
env:SHELL          : /bin/bash
env:LANG           : ru_RU.UTF-8
env:LANGUAGE       : ru:en_US

**Comment by AndreiBiziuk** [comment]
I can download the same video with yt-dlp without errors, 

>-> % yt-dlp YE7VzlLtp-4           
[youtube] YE7VzlLtp-4: Downloading webpage
[youtube] YE7VzlLtp-4: Downloading android player API JSON
[info] YE7VzlLtp-4: Downloading 1 format(s): 247+258
WARNING: Requested formats are incompatible for merge and will be merged into mkv
[download] Destination: Big Buck Bunny [YE7VzlLtp-4].f247.webm
[download]   1.2% of 40.03MiB at 515.83KiB/s ETA 01:18

> but when I try to play it with yewtube, I get errors.
> 
> -> % yt playurl YE7VzlLtp-4
> Big Buck Bunny
> ERROR: YE7VzlLtp-4: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 617, in extract
>     ie_result = self._real_extract(url)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 3575, in _real_extract
>     info['upload_date'] = strftime_or_none(self._extract_time_text(vpir, 'dateText')[0], '%Y%m%d')
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 732, in _extract_time_text
>     self._search_regex(
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 1184, in _search_regex
>     return next(g for g in mobj.groups() if g is not None)
> StopIteration
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1389, in wrapper
>     return func(self, *args, **kwargs)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1459, in __extract_info
>     ie_result = ie.extract(url)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 647, in extract
>     raise ExtractorError('An extractor error has occurred.', cause=e, video_id=self.get_temp_id(url))
> yt_dlp.utils.ExtractorError: YE7VzlLtp-4: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/main.py", line 69, in matchfunction
>     func(*matches)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/commands/play.py", line 142, in play_url
>     play(override, "1", "_")
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/commands/play.py", line 103, in play
>     g.PLAYER_OBJ.play(songlist, shuffle, repeat, override)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/player.py", line 73, in play
>     self.video, self.stream, self.override = stream_details(
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/player.py", line 322, in stream_details
>     streams.get(song, force=failcount, callback=screen.writestatus)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/streams.py", line 54, in get
>     ps = pafy.get_video_streams(ytid)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/pafy.py", line 38, in get_video_streams
>     info_dict = ydl.extract_info(ytid, download=False)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1380, in extract_info
>     return self.__extract_info(url, self.get_info_extractor(ie_key), download, extra_info, process)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1407, in wrapper
>     self.report_error(str(e), e.format_traceback())
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 939, in report_error
>     self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 879, in trouble
>     raise DownloadError(message, exc_info)
> yt_dlp.utils.DownloadError: ERROR: YE7VzlLtp-4: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U

**Comment by AndreiBiziuk** [comment]
Still getting error.

> andrei@andrei-arch [13:22:38] [~] 
> -> % pipx uninstall yewtube
> uninstalled yewtube! ✨ 🌟 ✨
> 
> andrei@andrei-arch [13:25:19] [~] 
> -> % pipx install git+https://github.com/iamtalhaasghar/yewtube.git
>   installed package yewtube 2.8.1, Python 3.10.2
>   These apps are now globally available
>     - yt
> done! ✨ 🌟 ✨
> 
> andrei@andrei-arch [13:26:12] [~] 
> -> % yt playurl YE7VzlLtp-4                                      
> Big Buck Bunny
> ERROR: YE7VzlLtp-4: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 617, in extract
>     ie_result = self._real_extract(url)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 3575, in _real_extract
>     info['upload_date'] = strftime_or_none(self._extract_time_text(vpir, 'dateText')[0], '%Y%m%d')
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 732, in _extract_time_text
>     self._search_regex(
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 1184, in _search_regex
>     return next(g for g in mobj.groups() if g is not None)
> StopIteration
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1389, in wrapper
>     return func(self, *args, **kwargs)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1459, in __extract_info
>     ie_result = ie.extract(url)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 647, in extract
>     raise ExtractorError('An extractor error has occurred.', cause=e, video_id=self.get_temp_id(url))
> yt_dlp.utils.ExtractorError: YE7VzlLtp-4: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/main.py", line 69, in matchfunction
>     func(*matches)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/commands/play.py", line 142, in play_url
>     play(override, "1", "_")
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/commands/play.py", line 103, in play
>     g.PLAYER_OBJ.play(songlist, shuffle, repeat, override)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/player.py", line 73, in play
>     self.video, self.stream, self.override = stream_details(
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/player.py", line 322, in stream_details
>     streams.get(song, force=failcount, callback=screen.writestatus)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/streams.py", line 54, in get
>     ps = pafy.get_video_streams(ytid)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/pafy.py", line 38, in get_video_streams
>     info_dict = ydl.extract_info(ytid, download=False)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1380, in extract_info
>     return self.__extract_info(url, self.get_info_extractor(ie_key), download, extra_info, process)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1407, in wrapper
>     self.report_error(str(e), e.format_traceback())
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 939, in report_error
>     self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
>   File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 879, in trouble
>     raise DownloadError(message, exc_info)
> yt_dlp.utils.DownloadError: ERROR: YE7VzlLtp-4: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U

**Comment by rachmadaniHaryono** [comment]
what about yewtube configuration? run `set` inside yewtube

it should be something similar like this

```shell
  Key                 Value
  order             : relevance
  user_order        :
  max_res           : 2160p
  player            : mpv
  playerargs        :
  encoder           : 0 [None]
  notifier          :
  checkupdate       : True
  show_player_keys  : True
  fullscreen        : False
  show_status       : True
  always_repeat     : False
  columns           : user date comments views
  ddir              : /mnt/ac54dceb-73a5-4f94-b52c-cb7a426c0f29/Downloads/mps
  overwrite         : True
  show_video        : False
  search_music      : False
  window_pos        :
  window_size       :
  download_command  :
  lookup_metadata   : True
  lastfm_username   :
  lastfm_password   :
  lastfm_api_key    :
  lastfm_api_secret :
  audio_format      : auto
  video_format      : auto
  pages             : 3
  autoplay          : False
  set_title         : True
  mpris             : True
  show_qrcode       : False
  history           : True
  input_history     : True
  vlc_dummy_interface : False

Enter set <key> <value> to change
Enter set all default to reset all
```

**Comment by AndreiBiziuk** [comment]
Getting the same error on default configuration `set all default`

```
Key                 Value                                                                                                     
  order             : relevance                 
  user_order        :                           
  max_res           : 2160p                     
  player            : vlc                       
  playerargs        :                           
  encoder           : 0 [None]                  
  notifier          :                           
  checkupdate       : True                      
  show_player_keys  : True                      
  fullscreen        : False                     
  show_status       : True                      
  always_repeat     : False                     
  columns           : date user                 
  ddir              : /home/andrei/Загрузки/mps 
  overwrite         : True                      
  show_video        : True                      
  search_music      : False                     
  window_pos        :                           
  window_size       :                           
  download_command  :                           
  lookup_metadata   : True                      
  lastfm_username   :                           
  lastfm_password   :                           
  lastfm_api_key    :                           
  lastfm_api_secret :                           
  audio_format      : auto                      
  video_format      : auto                      
  pages             : 3                         
  autoplay          : False                     
  set_title         : True                      
  mpris             : True                      
  show_qrcode       : False                     
  history           : True                      
  input_history     : True                      
  vlc_dummy_interface : False
```

**Comment by AndreiBiziuk** [comment]
Tried on different computer, getting errors as well.

**Comment by iamtalhaasghar** [comment]
> I've uninstalled yewtube
> 
> > pipx uninstall yewtube
> 
> and installed latest version
> 
> > pipx install git+https://github.com/iamtalhaasghar/yewtube.git
> 
> now almost every search or opening of playlist or playing video crashes with errors:
> 
> > ERROR: iuYxW_RiSaw: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> > Exception in thread Thread-2 (_preload):
> > Traceback (most recent call last):
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 617, in extract
> > ie_result = self._real_extract(url)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 3575, in _real_extract
> > info['upload_date'] = strftime_or_none(self._extract_time_text(vpir, 'dateText')[0], '%Y%m%d')
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 732, in _extract_time_text
> > self._search_regex(
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 1184, in _search_regex
> > return next(g for g in mobj.groups() if g is not None)
> > StopIteration
> > During handling of the above exception, another exception occurred:
> > Traceback (most recent call last):
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1389, in wrapper
> > return func(self, *args, **kwargs)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1459, in __extract_info
> > ie_result = ie.extract(url)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 647, in extract
> > raise ExtractorError('An extractor error has occurred.', cause=e, video_id=self.get_temp_id(url))
> > yt_dlp.utils.ExtractorError: iuYxW_RiSaw: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
> > During handling of the above exception, another exception occurred:
> > Traceback (most recent call last):
> > File "/usr/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
> > self.run()
> > File "/usr/lib/python3.10/threading.py", line 946, in run
> > self._target(*self._args, **self._kwargs)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/streams.py", line 189, in _preload
> > streamlist = get(song)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/streams.py", line 54, in get
> > ps = pafy.get_video_streams(ytid)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/mps_youtube/pafy.py", line 38, in get_video_streams
> > info_dict = ydl.extract_info(ytid, download=False)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1380, in extract_info
> > return self.__extract_info(url, self.get_info_extractor(ie_key), download, extra_info, process)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1407, in wrapper
> > self.report_error(str(e), e.format_traceback())
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 939, in report_error
> > self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
> > File "/home/andrei/.local/pipx/venvs/yewtube/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 879, in trouble
> > raise DownloadError(message, exc_info)
> > yt_dlp.utils.DownloadError: ERROR: iuYxW_RiSaw: An extractor error has occurred. (caused by StopIteration()); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U

Try `yt_dlp version     : 2022.02.04`. Seems like latest yt_dlp version has broken something.

**Comment by AndreiBiziuk** [comment]
> Try yt_dlp version : 2022.02.04. Seems like latest yt_dlp version has broken something.

Thank You for answering and patience )

Is it possible to do with pipx? pipx installs all libraries for yewtube in separate folder and automatically installs last version of yt-dlp. If I install specific version of yt-dlp with pipx it is installed separately and not used by yewtube.

I have installed yt-dlp version 2022.02.04
>pipx install yt-dlp==2022.02.04 --force

then I copied `~/.local/pipx/venvs/yt-dlp/lib/python3.10/site-packages/yt_dlp` folder into `~/.local/pipx/venvs/yewtube/lib/python3.10/site-packages` with overwrite and now yewtube is working again. Is there a better way to do this?
